documentation/contribute/add_a_dataset.qmd

```diff
@@ -113,8 +119,15 @@ platforms:
 ```
 :::
 
+For more parameter options, refer to @sec-parameters.
```

Comment:
this causes the website to show `section ...` this has no meaning to the reader. I would suggest to just add a link.
```suggestion
For more parameter options, refer to [Parameters](#parameters) section.
```